pontoon/teams/templates/teams/email_request_pretranslation.jinja

```diff
@@ -0,0 +1,9 @@
+{% autoescape off %}
+Please enable pretranslation for the following projects in {{ locale.name }} ({{ locale.code }}):
+{{ projects }}
+Requested by {{ user }}, {{ user_role }}:
+{{ user_url }}
+{% if locale.google_automl_model == '' %}
+IMPORTANT: Pretranslation is not enabled yet for {{ locale.name }} ({{ locale.code }}). Training a custom engine is required before enabling new projects.
```

Comment:
I replaced *this locale* with `{{ locale.name }} ({{ locale.code }})`.

And moved the paragraph to the bottom. On the very top, it's more visible, but also a little out of place, because one needs to read the text that follows in order to understand what is it about.